### PR TITLE
Add the doc about Freshers Selection

### PR DESCRIPTION
**Pull request by OrkoHunter**
**Write path(s) which will be affected by this Pull Request.**
- `/events/freshers-selection`

**Justify your changes or addition.**
Added a doc about recruiting new people. This doc needs to be updated every year. Added notes about the procedure and guidelines. Added positive and negative points of new members as well as red flags. Added FAQ and Archive section.

**Why do you think it should be documented?**
We should document because hiring can be very subjective. We need uniformity among all the interviewers across the years. However, the doc is just a guideline and must be updated every year.

The doc mentions our philosophy on a broad level. This information is not sensitive to the public.

### CODE REVIEW

**Comment by OrkoHunter** [comment]
I believe we can add more questions in the FAQ section.

**Comment by yashrsharma44** [comment]
I guess, we should add the more content in the selection docs regarding the **selection procedure for the previous years** also, where we had two slots for interview and this year, by using response sheets and then interview for selecting freshers?
This is important, as having **archive of previous year's selection** procedure can help us in improving our process of recruiting first years.

File: events/freshers-selection.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+### Positive points
+
+* Participated in KWoC or Summit
+* Follows KOSS, already in touch with any of its members
+* Tried tools/frameworks with a mindset to explore
+* Passion
+* Self-confidence
+
+### Negative points
```
Comment (Ayushk4) [inline events/freshers-selection.md]: Maybe we can make the difference more clear between `Negative points` and `Red Flags`.

**Comment by themousepotato** [comment]
Ping @OrkoHunter

**Comment by OrkoHunter** [comment]
Thank you @yashrsharma44 @Ayushk4 @xypnox for your comments! This is good to go.

**Pull request merged**